SingularityService/src/main/java/com/hubspot/singularity/config/TaskQuickLink.java

```diff
@@ -0,0 +1,35 @@
+package com.hubspot.singularity.config;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * TaskQuickLink defines a "quick-link" button to show on the
+ */
+public class TaskQuickLink {
+  /**
+   * Name of the button label in the SingularityUI interface
+   */
+  @JsonProperty
```

Comment:
`@JsonProperty` is unnecessary for immutable classes like this (just need `@JsonCreator` constructor + getters)